### ISSUE TITLE
FCOS: Fully Convolutional One-Stage Object Detection 

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#85]
# TL;DR
This study proposed a method,called FCOS,to detect the objects by regressing the correct distance of the bounding box from the each position ij of the feature map to without using anchors and by increasing the loss ratio of the center position of the BB. The proposed method showed a significant improvement compared with RetinaNet, which uses the same mechanism but with a bounding box.
![image](https://user-images.githubusercontent.com/24524018/92559229-1ee6e800-f2ab-11ea-9740-2b826ebf8c90.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/1904.01355

# Submission Dates(yyyy/mm/dd)

# Authors and institutions

# Methods

# Results

# Comments
